agent/Godeps/Godeps.json

```diff
@@ -309,31 +309,31 @@
 		},
 		{
 			"ImportPath": "golang.org/x/net/context",
-			"Rev": "f09c4662a0bd6bd8943ac7b4931e185df9471da4"
+			"Rev": "6c23252515492caf9b228a9d5cabcdbde29f7f82"
```

Comment:
Yep. First commit in the PR updates the golang.org dependencies. Should we avoid this?